packages/gluestick/src/cli/__tests__/logger.test.js

```diff
@@ -4,15 +4,25 @@ const loggerFactory = require('../logger');
 const colorScheme = require('../colorScheme');
 
 const logAndAssert = (message: string, shouldNotLog: string[], loggerInstance: Object) => {
-  Object.keys(loggerInstance).filter(key => key !== 'level').forEach((level: string): void => {
+  ['debug', 'info', 'warn', 'error', 'success'].forEach((level: string): void => {
```

Comment:
Can we take those values from somewhere? So if we add or remove one we do not need to come back to the test.